types/types.d.ts

```diff
@@ -380,7 +380,7 @@ export interface Page {
    * An example of handling `console` event:
    * 
    * ```js
-   * page.on('console', msg => {
```

Comment:
This file is autogenerated, the easiest way to update it is to run `npm run lint`.